clang/lib/CIR/CodeGen/CIRGenDecl.cpp

```diff
@@ -36,10 +36,10 @@ CIRGenFunction::AutoVarEmission
 CIRGenFunction::buildAutoVarAlloca(const VarDecl &D,
                                    mlir::OpBuilder::InsertPoint ip) {
   QualType Ty = D.getType();
-  // TODO: (|| Ty.getAddressSpace() == LangAS::opencl_private &&
-  //        getLangOpts().OpenCL))
   assert(!MissingFeatures::openCL());
-  assert(Ty.getAddressSpace() == LangAS::Default);
+  assert(
```

Comment:
This is safe. From the view of CIRGen, `opencl_private` is the same as `Default`. See also details in another comment.